src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke.exception;
```

Comment:
Like how you named each of your exceptions to make it clearer which exception is being returned!